misc/services/camel/de-metas-camel-shipping/src/main/resources/shipping.properties

```diff
@@ -8,9 +8,12 @@ shipmentCandidate.FMPXMLRESULT.DATABASE.NAME=databaseName
 local.file.output_path=/tmp/camel-test
 upload.endpoint.uri=log:upload-dummy
 
+siro.ftp.fileName.pattern=${file:name.noext}-${date:now:yyyy-MM-dd_HHmmssSSS}.${file:ext}
+siro.shipments.local.storage=file://tmp/camel-test/in/shipments?charset=utf-8&fileName={{siro.ftp.fileName.pattern}}
```

Comment:
wondering if we don't have a better path for this?

e.g. /tmp/metasfresh/shipment/siro